Dependencies.props

```diff
@@ -22,10 +22,14 @@
     <MoqVersion>4.12.0</MoqVersion>
     <PollySignedVersion>4.3.0</PollySignedVersion>
     <StackExchangeRedisStrongNameVersion>1.2.6</StackExchangeRedisStrongNameVersion>
-    <SystemCollectionsImmutableVersion>1.2.0</SystemCollectionsImmutableVersion>
+    <SystemCollectionsImmutableVersion>1.5.0</SystemCollectionsImmutableVersion>
```

Comment:
You can use `SystemCollectionsImmutableVersionCore` sinse it's already 1.5.0 this section is for dll's used in Omex repository so  I don't think that we should add any dependencies in this section.